okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -954,19 +959,38 @@ private void proxyInternal(Iterator<ModuleInstance> it,
     );
   }
 
+  static Buffer compressInternalResponse(RoutingContext ctx, Buffer respBuf) {
+    String acceptEncoding = ctx.request().getHeader("Accept-Encoding");
+    int sz = respBuf.length();
```

Comment:
Minor: maybe the variable name here `sz` could be more clear like `bufferLength` or just `size`.

Reply:
ok